agent/app/agent_unix.go

```diff
@@ -81,17 +81,15 @@ func (agent *ecsAgent) initializeTaskENIDependencies(state dockerstate.TaskEngin
 		return err, true
 	}
 
-	if agent.cfg.ShouldLoadPauseContainerTarball() {
```

Comment:
Since we will load the default pause container anyway, `ShouldLoadPauseContainerTarball` loses its meaning, the only consumer of this function is `platformString`, can you please name it to something that matches its meaning? Or you can just delete this function and move the logic into `platformString`